sktime/annotation/tests/test_igts.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""Tests for igts module."""
+
+import numpy as np
+import pytest
+
+from sktime.annotation.igts import IGTS, InformationGainSegmentation, entropy
```

Comment:
I am wondering why you can import `IGTS` and `entropy` here bcz you have in that file `__all__ = ["InformationGainSegmentation"]
`?

Reply:
`__all__` is only relevant when you do `from foo import *` but here I'm importing directly by names so it has no effect